README.md

```diff
@@ -1,6 +1,6 @@
 # Amazon ECS CLI v2
 
->⚠️ The ECS CLI v2 is currently under active development and is not suitable for production workloads. 
+>⚠️ Before you get started please note that this feature is in preview and the intention is to ensure it meets your requirements and give us any feedback on your use case. Please do not run production workloads till we announce the general availability of this feature. Using the instructions and assets in this repository folder is governed as a preview program under the AWS Service Terms: https://aws.amazon.com/service-terms/ . 
```

Comment:
Maybe it's just me, but I find the first sentence confusing.

Reply:
It's our "official" line from the containers roadmap. App Sec gave it to me to add.